.flake8

```diff
@@ -8,8 +8,8 @@ ignore = E203, E501, W503,  # handled by black
 exclude =
     venv
     venv
-max-complexity = 7
-max-cognitive-complexity = 5
+max-complexity = 10
+max-cognitive-complexity = 10
```

Comment:
I expect that you did these changes in master and later merge them to your branches.